app/UuidBase.php

```diff
@@ -12,4 +13,17 @@ class UuidBase extends Eloquent
      * @var bool
      */
     public $incrementing = false;
+
+
+    /**
+     * Boot function from laravel.
+     */
+    protected static function boot()
```

Comment:
I don't know if this is necessary, but I wasn't able to get UUID ids to generate for submissions or acceptances without it. If there's another way, please let me know.

Reply:
In the current setup, this should go in `app/modelEvents.php`. I'm OK if we move it to here as long as we also cover the deleting event in talk, and then delete the modelEvents file.

Reply:
This is still open -- please either move this to modelEvents, or delete that file, so we're not double-binding this on those.